mods/cnc/maps/shellmap/shellmap.lua

```diff
@@ -1,3 +1,362 @@
+CommandoTeam = {"rmbo", "rmbo"}
+GDIAttackTeam = {"e1", "e1", "e2"}
+GDIInfantryTeam = {"e1", "e1", "e1", "e2", "e2"}
+GDIVehicleTeam = {"jeep","jeep"}
+NodInfantry =
+	{
+		{{"e1", "e1", "e3"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e1", "e1", "e1", "e3", "e3"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e4", "e4"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e1", "e1", "e1", "e1", "e3", "e3", "e4", "e4" },function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end}
+	}
+NodVehicles =
+	{
+		{{"ltnk", "ltnk", "ltnk"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"bike", "bike", "bike"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"bike", "bike"},function(a)
+			a.AttackMove(HandBuild.Location+CVec.New(3,5))
+			a.AttackMove(MCVDeploy.Location)
+		 end},
+		{{"bggy", "bggy", "bggy"},function(a)
+			a.AttackMove(HandBuild.Location+CVec.New(3,5))
+			a.AttackMove(MCVDeploy.Location)
+		 end}
+	}
+OutpostInfantry =
+	{
+		{{"e1", "e1", "e3"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e1", "e1", "e1", "e3", "e3"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e4", "e4"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e1", "e1", "e1", "e1", "e3", "e3", "e4", "e4" }, function(a) a.AttackMove(MCVDeploy.Location) end }
+	}
+
+LSTREntryPath = {LSTRStart.Location, LSTRMove.Location, LSTRLand.Location}
+LSTMEntryPath = {LSTMStart.Location, LSTMMove.Location, LSTMLand.Location}
+LSTLEntryPath = {LSTLStart.Location, LSTLMove.Location, LSTLLand.Location}
+
+LSTRExitPath = {CPos.New(LSTRMove.Location.X,LSTLStart.Location.Y), CPos.New(8,LSTLStart.Location.Y)}
+LSTMExitPath = {LSTMMove.Location, CPos.New(8,LSTMStart.Location.Y)}
+LSTLExitPath = {CPos.New(LSTLMove.Location.X,LSTRStart.Location.Y), CPos.New(8,LSTRStart.Location.Y)}
+
+--Repeated Functions START
+UnloadLST = function(lst,cargo)
+	Utils.Do(cargo, function()
+		local a = lst.UnloadPassenger()
+		a.Teleport(lst.Location)
+		a.IsInWorld = true
+		a.Move(lst.Location - CVec.New(0, 1))
+	end)
+end
+
+PlayMusic = function()
+	Media.PlayMusic("map1", PlayMusic)
+end
+
+StayInLocation = function(a,loc,dist)
+	if not a.IsDead then
+		local vector = CVec.New(a.Location.X-loc.X,a.Location.Y-loc.Y)
+		if math.abs(vector.X) > dist or math.abs(vector.Y) > dist then
+			a.Stop()
+			a.AttackMove(loc)
+			Trigger.AfterDelay(250, function() StayInLocation(a,loc,dist) end)
```

Comment:
Please use the `DateTime.Seconds` and/or `DateTime.Minutes` functions instead of hard-coded numbers for better readability.